### ISSUE TITLE
Using Git Bash rather than GitHub GUI

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/CORRECTIONS#247]
This comment describes a problem I've encountered working with GitHub, and a solution that seems to resolve the problem.

### The Problem:  
Today I was working with the [MWderivations](https://github.com/funderburkjim/MWderivations) repository.   I wanted to commit a large chunk (about 40MB) comprising the 'step3' directory therein.

Using the GitHub GUI interface, I clicked the 'changes' tab, entered brief 'Summary' comment, and then clicked 'Commit to master'.  So far so good.  But now, after about 20-30 minutes,  this commit action had still not completed.  I was left in limbo, not knowing what to think.   At some point I just closed the GitHub gui.

This same scenario has played in working with several other repositories.   It appears to be some bug in the GitHub GUI.

### The Solution: use Git Bash

Fortunately, the problem doesn't occur when three simple command-line commands are used.
After opening GitBash terminal, I changed to the MWderivations directory.  This had been properly
set up earlier when I cloned a prior version of MWderivations repository at GitHub; this cloning had
been done with the GitHub GUI.

Here are the three steps:
* git add .
  * that '.' means to add *all* the files that need to be added to the Git *staging area*.
  * This took only a few seconds
* git commit -m "step3"
   * -m means 'mesage', and my message here was just "step3"
   * This also took only a few seconds.
* git push origin master
  * *origin* here means the GitHub remote.  If I do a 'git config -l' command later, one of the
     configuration settings shows as 
     * remote.origin.url=https://github.com/funderburkjim/MWderivations.git
  * *master* here means the local repository branch, I think.
  * I was prompted for my Github username and password.
  * Then, all the changes just commited were pushed up to GitHub.  This took 3-5 minutes, since
     I have very slow upload bandwidth.

Going via the browser to the MWderivations repository at GitHub,  I see that all the changes are 
really there.

This note is marked as documentation, since it may prove of use to others.

**Comment by drdhaval2785** [comment]
I have also encountered this issue before. I have already shifted to git
bash.

My experience says that the delay in creating commits via github GUI is
because with every commit, it tries to prepare a display showig differences
in this commit. For large files, this proves costly memory wise.

Only drawback in git bash is I cannot close an issue via commit. E.g. in
github GUI, I can type 'Issue 124 closed' as commit message, and when I
sync, the issue gets closed. I am unable to get the same result via git
bash. So I have to manually close the issue.

My solution is - for smaller changes in code which close an issue, use
github GUI.
For larger MB files or adding some bulky data, use git bash.
